dataactcore/models/domainModels.py

```diff
@@ -25,4 +25,14 @@ class ObjectClass(Base):
     __tablename__ = "object_class"
     object_class_id = Column(Integer, primary_key=True)
     object_class_code = Column(Text,nullable=False,index=True,unique=True)
-    object_class_name = Column(Text)
\ No newline at end of file
+    object_class_name = Column(Text)
+
+class ProgramActivity(Base):
+    __tablename__ = "program_activity"
+    program_activity_id = Column(Integer, primary_key=True)
+    budget_year = Column(Text,nullable=False)
+    agency_id = Column(Text,nullable=False)
```

Comment:
Is the agency_id here the same as a CGAC code? If so, should we consider naming this column cgac_code to be consistent with the CGAC model? I'm honestly a bit torn on how to refer to agencies throughout the broker, since CGAC doesn't seen to be common parlance. That said, using it consistently would remove any ambiguity around the department/agency/bureau discussion.